distributions/Dist_hk1764.py

```diff
@@ -0,0 +1,47 @@
+
+import numpy as np
+from base_distribution import BaseDistribution
+
+class Dist_hk1764(BaseDistribution):
+	def __init__(self):
+		self.f_max = 1
+		self.x_min = -1
+		self.x_max = 1
+
+
+	def pdf(self, x):
+		"""This is your PDF"""
+		return np.abs((x**2-x**3)/x)
+
+	def mean(self):
+		"""This is the mean of the PDF"""
+		return -.41
```

Comment:
There's nothing wrong with the PDF. The issue is that the mean and std should not just be numbers, but should be based on equations you derived using calculus and the definition of the mean and standard deviation of a distribution.